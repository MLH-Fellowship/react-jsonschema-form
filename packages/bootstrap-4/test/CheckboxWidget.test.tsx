// TODO: we need to use the mocked props for widget props ,
// Helper function has been created in https://github.com/MLH-Fellowship/react-jsonschema-form/pull/37
// wait for the merging of the mentioned PR.
