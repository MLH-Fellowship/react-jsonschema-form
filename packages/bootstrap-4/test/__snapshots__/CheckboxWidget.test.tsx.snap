// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckboxWidget simple 1`] = `
<div
  className="checkbox disabled"
>
  <h2
    className="mt-5"
  >
    Some simple label
  </h2>
  <label>
    <input
      autoFocus={true}
      checked="value"
      className="form-control"
      disabled={true}
      id="_id"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      required={true}
      type="checkbox"
    />
  </label>
</div>
`;
